### PR TITLE
Editorial updates to the README

### PR DESCRIPTION
**Pull request by supear**
These updates include a reference to the current status of multi-tenancy support.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,61 +65,62 @@ A file server that uses the network file system (NFS) protocol is the minimum re
...
+The playbook uses Ansible variables for configuration. SAS recommends that you encrypt both this file and the other configuration files (sitedefault, kubeconfig, and tfstate) using [Ansible Vault](https://docs.ansible.com/ansible/latest/user_guide/vault.html).
 
-#### Ansible Vars
+#### Ansible Vars File
 
-The ansible vars file is the main configuration file. Create a file named ansible-vars.yaml to customize any input variable value. Example variable definition files are provided in the ./examples folder. For more details on the supported variables refer to [CONFIG-VARS.md](docs/CONFIG-VARS.md).
+The Ansible vars.yaml file is the main configuration file. Create a file named ansible-vars.yaml to specify values for any input variables. Example variable definition files are provided in the `./examples` folder. For more details on the supported variables, refer to [CONFIG-VARS.md](docs/CONFIG-VARS.md).
 
-#### Sitedefault
+#### (Optional) Sitedefault File
```
Comment (thpang) [inline README.md:111]: Do you normally have Optional before the heading? I usually see this after like

#### Sitedefault File (Optional)

File: README.md [hunk 2]
```diff
@@ -127,7 +130,7 @@ This following information is parsed from the integration:
 - RWX Filestore
   - V4_CFG_RWX_FILESTORE_ENDPOINT
   - V4_CFG_RWX_FILESTORE_PATH
-- JumpBox
+- Jump VM
```
Comment (thpang) [inline README.md:133]: We do use `JumpBox` instead of `Jump VM` This value should not change.

**Review by thpang — CHANGES_REQUESTED** [review]
Just a few requested changes.

File: README.md (hunk 1 above)
Comment (supear) [reply README.md:111]: We place it before the heading in the docs. For ex., see https://go.documentation.sas.com/doc/en/calcdc/3.5/dplyml0phy0lax/n0w37tlhoww4wmn1s1w563uo23mn.htm

**Review by thpang — APPROVED** [review]
Thanks for the update and clarification.

**Pull request merged**